netlify/_redirects_v3.21

```diff
@@ -0,0 +1,254 @@
+# These rules are specific to docs.projectcalico.org site, backed by release-v3.21 branch, which is the last production version hosted at docs.projectcalico.org.
```

Comment:
For my understanding these rules never need to be updated at all? (of course doesn't apply to any bug fixes)

Reply:
yep, these will be static. All the new versions will be server from the new domain. Couple of moving versions we track in this are `latest` at `/manifests` and `master/nightly`, which are pointed to their dedicated sites.